packages/backend/src/tokens/jwt/assertions.test.ts

```diff
@@ -259,7 +259,16 @@ export default (QUnit: QUnit) => {
     });
   });
 
-  module('assertExpirationClaim(exp, clockSkewInMs)', () => {
+  module('assertExpirationClaim(exp, clockSkewInMs)', hooks => {
+    let fakeClock;
+    hooks.beforeEach(() => {
+      fakeClock = sinon.useFakeTimers();
+    });
+    hooks.afterEach(() => {
+      fakeClock.restore();
+      sinon.restore();
+    });
```

Comment:
Just curious - is this pattern common enough for us to consider writing an abstraction?

Reply:
Let's keep that as a note and if there is another occurrence. I will refactor it as a testHelper.